static/js/components/ProfileProgressControls.js

```diff
@@ -55,9 +63,9 @@ export default class ProfileProgressControls extends React.Component {
     if (nextUrl) {
       nextButton = <button
         role="button"
-        className="mdl-button next"
-        onClick={this.saveAndContinue}>
-        <span>{nextBtnLabel}</span>
+        className={`mdl-button next ${disabled ? 'disabled-with-spinner' : ''}`}
+        onClick={disabled ? undefined : this.saveAndContinue}>
+        {nextBtnLabel}
```

Comment:
could this just be

```js
{disabled ? <Spinner singleColor /> : nextBtnLabel }
```

I often find having the control flow in the JSX to be more readable than having something declared above, that's just my preference though I think.